app/models/vmdb_metric.rb

```diff
@@ -31,4 +32,9 @@ def self.rollup_metrics(resource, _interval_name, rollup_date)
     # Create new daily record...
     metric.update_attributes(:rows => rows, :size => size, :wasted_bytes => wasted_bytes, :percent_bloat => percent_bloat)
   end
+
+  def self.reindex_table_name
+    "metrics_#{DateTime.now.utc.hour}"
```

Comment:
@carbonin is this the appropriate class for this, and am I understanding it [correctly](https://github.com/ManageIQ/manageiq-appliance/blob/ce0af6e01373a35ba57ddcb38d5eacb81261e5d6/LINK/usr/bin/hourly_reindex_metrics_tables#L5) that it should be performed on `metrics_<hour_of_day>`?

Reply:
I think it's the next hour.

Reply:
Also I think this should be on `Metric` not `VmdbMetric`